### PR TITLE
Integrate PostCSS-plugin for RTL-adaptivity

### PR DESCRIPTION
**Pull request by sanabhass**
## PR Checklist

Please check all that apply to this PR using "x":

- [ X ] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [ X ] I have checked that this PR does not introduce a breaking change
- [  ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [  X ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number: [#4430](https://github.com/lbryio/lbry-desktop/issues/4430)

## What is the current behavior?
Right-to-left languages are not properly supported by the application. Neither direction switch nor CSS adaptions are implemented yet.

## What is the new behavior?
PostCSS-plugin  will generates RTL rules with flipped properties and generate one file for both directions.
Manage direction by switching between dir="ltr" and dir="rtl" on <html> element

### CODE REVIEW

**Comment by seanyesmunt** [comment]
Thanks! Will check this out after our latest app version is released

**Comment by seanyesmunt** [comment]
@sanabhass I'm not sure what is wrong, but I tried running this branch and this is what it looks like:
Screen Shot 2020-07-13 at 1 14 29 PM

**Comment by sanabhass** [comment]
@seanyesmunt sorry for the late reply, I was busy. 
In the previous PR  I forgot to add the default direction (which is ltr) in the html frame. I added it and correspondent css style is added it now. To load rtl css style you can switch the dir attribute in the html frame to rtl `<html dir="rtl">`.

**Comment by seanyesmunt** [comment]
No problem! I'll get to testing this later today or tomorrow morning!

File: static/index-web.html [hunk 1]
```diff
@@ -64,6 +64,13 @@
     </style>
 
     <!-- VARIABLE_HEAD_BEGIN -->
+    <title>lbry.tv</title>
```
Comment (seanyesmunt) [inline static/index-web.html:67]: Can you remove these additions?

**Pull request merged**